remoulade/api/main.py

```diff
@@ -26,8 +27,8 @@ def get_states():
             item for item in data if chr(0).join([str(item[k]) for k in keys if item.get(k)]).lower().find(value) >= 0
         ]
 
-    if args.get("sort_column"):
-        reverse = args.get("sort_direction") == "desc"
+        if args.get("sort_column"):
```

Comment:
What is this ? You have to read more carefully the diff on github before assign it to me. And ask yourself is this really necessary or is this the result of my process of programming.
There should be no useless lines.